src/qn/flux.jl

```diff
@@ -7,47 +6,75 @@ Returns the flux of the ITensor.
 
 If the ITensor is empty or it has no QNs, returns `nothing`.
 """
-function flux(T::ITensor)
-  return flux(tensor(T))
-end
+flux(T::ITensor, args...) = flux(tensor(T), args...)
 
-function checkflux(T::ITensor, flux_check)
-  return checkflux(tensor(T), flux_check)
-end
+"""
+    checkflux(T::ITensor)
 
-function checkflux(T::ITensor)
-  return checkflux(tensor(T))
-end
```

Comment:
Here I was just turning some multi-line functions into single-line expressions, but they are the same code.